MapboxVision/src/main/java/com/mapbox/vision/utils/prefs/SessionPrefs.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.vision.utils.prefs
+
+import com.mapbox.vision.mobile.core.utils.preferences.LongAdapter
+import com.mapbox.vision.mobile.core.utils.preferences.Preference
+import com.mapbox.vision.mobile.core.utils.preferences.PreferenceModel
+
+interface SessionPrefs {
+    val sessionStartMillis: Preference<Long>
+    val bytesSentPerSession: Preference<Long>
+
+    object Impl : SessionPrefs, PreferenceModel("session_prefs") {
```

Comment:
we already have 2 Sync modules that use 'TotalBytsCounter' and they should have different ones with different `SessionPrefs`. So in my opinion `SessionPref` -> `TotalBytesCounterStorage` and `TelemertyImpl : TotalBytesCounterStorage`

Reply:
second option: have only one `TotalBytesCounterStorage` and `mpl: TotalBytesCounterStorage` but make `Impl` not `object` but just `class` with  `constructor (name: String)` and use `name` as `prefix` for _SharePreference_ strorage

Reply:
done